scripts/uppmax_ngi_reports.sh

```diff
@@ -22,7 +22,7 @@ function make_report {
       fn=${f%.md}
       echo "Converting ${f}.."
       PD_DIR=${ngi_reports_dir}/data/pandoc_templates
-      ${PD_DIR}/pandoc --standalone --section-divs ${fn}.md -o ${fn}.html --template=${PD_DIR}/html_pandoc.html --default-image-extension=png --filter ${PD_DIR}/pandoc_filters.py -V template_dir=${PD_DIR}/ 
+      ${PD_DIR}/pandoc --standalone --section-divs ${fn}.md -o ${fn}.html --template=${PD_DIR}/html_pandoc.html --default-image-extension=png --filter ${PD_DIR}/pandoc_filters.py -V template_dir=${PD_DIR}/
```

Comment:
I guess this was auto formatting from your editor, but I think that these bash scripts can also be deleted. All they do is run pandoc I think.

Reply:
That's the next step! ✂️